packages/site/src/layout/SearchBox.module.css

```diff
@@ -1,5 +1,6 @@
 .searchBoxResults {
     width: 100%;
+    height: 600px;
```

Comment:
Sort of a magic number but it feels pretty comfortable; shows 2 rows plus a third row to start, giving a clear indication of scrollability